src/App.js

```diff
@@ -3,9 +3,16 @@ import { hide } from 'react-native-bootsplash';
 import { GestureHandlerRootView } from 'react-native-gesture-handler';
```

Comment:
We should update this lib as well, because they added support for RN 0.70 in [version 2.6.0](https://github.com/software-mansion/react-native-gesture-handler/releases/tag/2.6.0)